### PR TITLE
chore: add cf deployment

### PR DESCRIPTION
**Pull request by Lykhoyda**
## Description
- Add widget package deployment during the push to the main and when opening a new PR
- Separate deployment of the widget package when a new version is released
- Add deployment for react-widget on push and opening a PR
- Add production deployment for react-widget on a new release

Closes: #28

### CODE REVIEW

**Comment by wainola** [comment]
@Lykhoyda how do we envision to deploy the react package?

**Review by mpetrunic — CHANGES_REQUESTED** [review]
We don't need prod deployment (we are building library here), we just need deploy previews.
It's probably best to rebase against #58

File: packages/react/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "license": "LGPL-3.0-or-later",
   "type": "module",
   "dependencies": {
-    "@lit-labs/react": "2.0.3"
+    "@lit-labs/react": "2.1.2"
```
Comment (Lykhoyda) [inline packages/react/package.json:10]: The changes done to trigger the react workflow

**Comment by Lykhoyda** [comment]
> @Lykhoyda how do we envision to deploy the react package? I have this question after reading the deploy files for the react package

@wainola the deployment will serve the static file like index.html. The react project is empty for now, so it's just a preparation of the code for future code deployments.

**Review by wainola — APPROVED** [review]

**Review by wainola — APPROVED** [review]

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**